modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -37,17 +37,7 @@ For additional details on {productname} 6.0 changes, see xref:6.0-release-notes-
 
 Previously, when loading a requested resource, the `AddOnManager` passed a `success` or `failure` callback to report the status of the request. In {productname} 6, `AddOnManager` reports this status by returning a `Promise`, with a value of either `resolve` or `reject`.
 
-=== `closeButton` and `timeout`
-
-Previously, using `closeButton` to disable the close button presented as part of a _Notification_ required a `timeout` value to also be set.
-
-In {productname} 6, `closeButton` shows or hides the close button regardless of a `timeout` value being set or not set.
-
-Equivalently, setting a `timeout` value sets how long the notification presents before automatically closing, regardless of whether a close button is displayed or not.
-
-NOTE: This does not change the behaviour of existing _Notifications_. An `editor.notificationManager` with both `timeout: 5000` and `closeButton: false` set will still present a notification without a close button that automatically closes after displaying for 5 seconds.
-
-=== `disabled` functions renamed in all {productname} APIs
+=== `disabled` wording renamed in all {productname} APIs
```

Comment:
```suggestion
=== `disabled` property renamed in all {productname} APIs
```

Reply:
```suggestion
=== `disabled` renamed in all {productname} APIs
```
not quite sure on the context, but this honestly might be easier?

Reply:
I used wording both here and below, I think the headings should refer to the wording